src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -32,5 +34,62 @@ export class OverflowMenuPane extends Dialog {
 		 * so we need to add some compensation)
 		 */
 		this.addGap["bottom"] = pos => position.addOffset(pos, -20, 60);
+
+		setTimeout(() => {
+			this.listItems()[0].focus();
+		});
+	}
+
+	@HostListener("keydown", ["$event"])
+	hostkeys(event: KeyboardEvent) {
+		this.escapeClose(event);
+		const listItems = this.listItems();
+
+		switch (event.key) {
+			case "ArrowDown": {
+				event.preventDefault();
+				if (!isFocusInLastItem(event, listItems))  {
+					const index = listItems.findIndex(item => item === event.target);
+					listItems[index + 1].focus();
```

Comment:
We recently got [an issue against focusing on hidden dropdown item](https://github.com/IBM/carbon-components/issues/1107) (though not overflow menu) - Just in case you want to take this into consideration.

Reply:
Can overflow menu even have hidden items?

Reply:
Good question, we don't have a definition wrt one vs. other. That said, another approach is documenting "no support for hidden overflow menu item".